package.json

```diff
@@ -28,7 +28,10 @@
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-react": "^7.12.5",
     "babel-loader": "^8.2.1",
+    "grommet": "^2.16.0",
+    "grommet-icons": "^4.5.0",
     "react-hot-loader": "^4.13.0",
+    "styled-components": "^5.2.1",
```

Comment:
This doesn't seem to be used anywhere. Is it a mistake to include it in the pull request?

Reply:
According to the docs this should be installed:

npm install grommet grommet-icons styled-components --save

But maybe this is only necessary when working with dark mode, making css customization etc?

Reply:
I personally like that this library makes use of styled-components, it's a great way of componetizing style in my opinion. If you're not familiar with it @kristofferlarberg, I would suggest styling a couple of the components to see how it works, for instance make an OkButton and a CancelButton both based on Button.

Reply:
Documentation can be found here https://styled-components.com/